### PR TITLE
Reformatting PostgreSQL grammar&lexer using antlr formatter

### PR DESCRIPTION
**Pull request by OleksiiKovalov**
**What does this PR do?**
reformatting grammar&lexer using antlr formatter ( https://github.com/antlr/Antlr4Formatter ) to follow community approach
removing extra parenthesys
**Why?**
now grammar looks uniform with other grammars
**How is it checked?**
Tested using unit tests over the tests scripts from the original PostgreSQL repository

### CODE REVIEW

File: sql/postgresql/PostgreSQLLexer.g4 [hunk 1]
```diff
@@ -27,90 +28,168 @@ https://github.com/tunnelvisionlabs/antlr4-grammar-postgresql/blob/master/src/co
...
 /* Reference:
  * http://www.postgresql.org/docs/9.3/static/sql-syntax-lexical.html
  */
 
-options { language = CSharp; }
 
-@header {
+
+options { language = CSharp; }
+@ header
```
Comment (KvanTTT) [inline sql/postgresql/PostgreSQLLexer.g4:39]: It looks like more clear without whitespace:

```suggestion
@header
```

File: sql/postgresql/PostgreSQLLexer.g4 [hunk 2]
```diff
@@ -27,90 +28,168 @@ https://github.com/tunnelvisionlabs/antlr4-grammar-postgresql/blob/master/src/co
...
+PERCENT
+   : '%'
+   ;
+
+PARAM
+   : '$' ([0-9])+
+   ;
+   //
+   
+   // OPERATORS (?4.1.3)
```
Comment (KvanTTT) [inline sql/postgresql/PostgreSQLLexer.g4:174]: I suggest removing excess indents like this one. Also `?` looks weird here (probably it's here due to incorrect encoding conversion):

```suggestion
// OPERATORS (4.1.3)
```

File: sql/postgresql/PostgreSQLLexer.g4 [hunk 3]
```diff
@@ -121,638 +200,2154 @@ Operator
...
+CONSTRAINT
+   : C O N S T R A I N T
+   ;
+
+CREATE
+   : C R E A T E
+   ;
+
+CURRENT_CATALOG
+   : C U R R E N T [_] C A T A L O G
```
Comment (KvanTTT) [inline sql/postgresql/PostgreSQLLexer.g4:412]: I suggest using quotes for a single symbol:

```suggestion
   : C U R R E N T '_' C A T A L O G
```

**Review by KvanTTT — COMMENTED** [review]
Just for curious: have you used a tool for automatic formatting?

**Comment by OleksiiKovalov** [comment]
>>Just for curious: have you used a tool for automatic formatting?
yup,  https://github.com/antlr/Antlr4Formatter (looks like it has some issues in unit tests)

**Pull request merged**